apps/howoldami/howoldami.star

```diff
@@ -0,0 +1,147 @@
+                            main_align = "space_between",
+                            expanded = True,
+                            children = [
+                                render.Text("Secs:", font = font, color = "#ff0000"),
+                                render.Animation(
+                                    children = secondList,
+                                ),
+                            ],
+                        ),
+                        pad = (0, 0, 0, 1),
+                    ),
+                    render.Padding(
+                        child = render.Row(
+                            main_align = "space_between",
+                            expanded = True,
+                            children = [
+                                render.Text("Mins:", font = font, color = "#f2c57c"),
+                                render.Animation(
+                                    children = minuteList,
+                                ),
+                            ],
+                        ),
+                        pad = (0, 0, 0, 1),
+                    ),
+                    render.Padding(
+                        child = render.Row(
+                            main_align = "space_between",
+                            expanded = True,
+                            children = [
+                                render.Text("Hours:", font = font, color = "#55ef67"),
+                                render.Animation(
+                                    children = hourList,
+                                ),
+                            ],
+                        ),
+                        pad = (0, 0, 0, 1),
+                    ),
+                    render.Padding(
+                        child = render.Row(
+                            main_align = "space_between",
+                            expanded = True,
+                            children = [
+                                render.Text("Days:", font = font, color = "#00a9aa"),
+                                render.Text("%d" % float(days), font = font, color = "#00a9aa"),
+                            ],
+                        ),
+                        pad = (0, 0, 0, 1),
+                    ),
+                ],
+            ),
+        ),
+    )
+
+def get_schema():
+    options = [
+        schema.Option(display = "US/Central", value = "US/Central"),
+        schema.Option(display = "US/Eastern", value = "US/Eastern"),
+    ]
+
+    return schema.Schema(
+        version = "1",
+        fields = [
+            schema.Text(
+                id = "birthdate",
+                name = "Birthdate",
+                desc = "Birthdate (format: 2020-01-01)",
+                icon = "user",
+            ),
+            schema.Text(
+                id = "birthtime",
+                name = "Birthtime",
+                desc = "Birthtime (format: 13:00:00)",
+                icon = "user",
+            ),
+            schema.Dropdown(
```

Comment:
I would suggest using the (kinda magic) `$tz` config parameter. It's provided automatically and will match the timezone of the location the user has selected for their Tidbyt device.

In cases where it's important to make timezone configurable per app installation, the simplest way is to use the `Location` schema field. It allows the user to pick any location in the world, and the associated timezone can be found in the config data passed to the app. But for this app I bet `$tz` will do just fine.

Reply:
Also good to use here - updated.